src/main.rs

```diff
@@ -503,6 +512,11 @@ fn run() -> Result<(), failure::Error> {
 
         let method = HTTPMethod::from_str(matches.value_of("method").unwrap_or("get"))?;
 
+        let default_url = "https://example.com";
+
+        let url = Url::parse(matches.value_of("url").unwrap_or(default_url))
+            .unwrap_or(Url::parse(default_url)?);
```

Comment:
We should change this `.unwrap_or` to be a `?` so that it fails if the url fails to parse. That way if the user passes something like `--url invalid-string`, they can fix their mistake without finding out that they input something incorrect before wrangler starts to rebuild

Reply:
I agree, have added validation here (including scheme validation as either `http` or `https`)